### ISSUE TITLE
Make it easier to work with chemical reaction recipes! 

### ISSUE THREAD

**Issue by BoobaPog** [petrolpark/Destroy#311]
I am trying to include the 'Destroy' in my mod pack. However, dealing with reaction recipes is quite challenging. Specifically, I am struggling to get anything to work with 'destroy:reaction' and have no idea how to do it. As someone who lacks insight into this mod, it would be helpful to have tools that make it easier to manipulate or add new chemical reactions for interactions with other mods.

**Comment by petrolpark** [comment]
There is no recipe type `destroy:reaction`. Reactions are not recipes. Once #299 is merged in, it will be possible to register molecules and simple reactions with KubeJS, but for now the only way is with an addon mod. The way this is done with an addon mod will be made simpler in future.

**Comment by BoobaPog** [comment]
> There is no recipe type `destroy:reaction`. Reactions are not recipes.

Okay, that is probably why I could not get it to work. I must point out that you are mistaken. It is possible that you accidentally registered 'destroy:reaction' as a recipe, and that might have caused my confusion. I used KubeJS to look up all recipes, and I found that chemical reactions are registered under the recipe type:
<recipetype:destroy:reaction>, and
destroy:reaction [com.petrolpark.destroy.recipe.DestroyRecipeTypes$1]
